### PR TITLE
Update docs: Alert policies

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>
The images are wrong due to the removal of some components. I removed and updated the figures and added some descriptions. Please take a look, thanks.
/assign @Patrick-LuoYu @Felixnoo

### CODE REVIEW

File: content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md [hunk 1]
```diff
@@ -52,19 +48,24 @@ KubeSphere also has built-in policies which will trigger alerts if conditions de
...
+Click the name of an alerting policy on the **Alerting Policies** page to see its detail information, including alerting rules and alerting messages. You can also see the rule expression which is based on the template you use when creating the alerting policy.
 
 Under **Monitoring**, the **Alert Monitoring** chart shows the actual usage or amount of resources over time. **Notification Settings** displays the customized message you set in notifications.
 
-![alert-detail](/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/alert-detail.png)
+![alerting-policy-details-page](/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/alerting-policy-details-page.png)
+
+{{< notice note >}}
+
+You can click <img src="/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/drop-down-list.png" width='20' /> on the upper-right corner to select or custom a time range for the alert monitoring chart.
```
Comment (Felixnoo) [inline content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md:67]: `on` -> `in`

File: content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md [hunk 2]
```diff
@@ -52,19 +48,24 @@ KubeSphere also has built-in policies which will trigger alerts if conditions de
...
+
+You can also click <img src="/images/docs/zh-cn/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy-node-level/refresh.png" width='25' /> on the upper-right corner to manually refresh the alert monitoring chart.
```
Comment (Felixnoo) [inline content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md:69]: `on` -> `in`

**Review by Felixnoo — CHANGES_REQUESTED** [review]

File: content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md (hunk 1 above)
Comment (serenashe) [reply content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md]: Changed. I did not notice this before. Thanks very much.

**Pull request merged**